project/Build.scala

```diff
@@ -1303,6 +1303,16 @@ object Build {
       }
     }
 
+  def staticAssets = Def.task {
```

Comment:
Since this is a task (which performs side-effects), I suggest using a verb as a name, like `generateStaticAssets`.